rust-runtime/aws-smithy-http-server/examples/pokemon_service/src/lib.rs

```diff
@@ -125,6 +125,7 @@ impl Default for State {
 }
 
 /// Retrieves information about a Pokémon species.
+#[tracing::instrument(name = "GetPokemonSpecies", skip(input, state))]
```

Comment:
I think the framework should be the one creating the span at the callsite, with a standardized name, and taking into account not logging `sensitive` data https://github.com/awslabs/smithy-rs/pull/1314/files#diff-415c36a8d1ab7f3fe22cd864e7b505f40a55b90da5140becee779b323ba25b45R93

Reply:
You are right, we should do it in the code generated part. I am going to remove this.